packages/dd-trace/src/appsec/reporter.js

```diff
@@ -444,7 +432,7 @@ function reportAttributes (attributes) {
 
   const tags = {}
   for (let [tag, value] of Object.entries(attributes)) {
-    if (!isFingerprintAttribute(tag)) {
+    if (isSchemaAttribute(tag)) {
```

Comment:
Conditional has been flipped, to apply the special format (gizp + base64 encoding) only to schema attributes (aka derivatives)